src/index.html

```diff
@@ -4,19 +4,108 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="is-active nav__link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav__item">
```

<img width="1109" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/e09fb295-feba-476e-9f34-6970b972e3e8">

The same in all places